chain/checkpointing/storage.go

```diff
@@ -0,0 +1,36 @@
+package checkpointing
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"os"
+
+	"github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/credentials"
+)
+
+func StoreConfig() {
+	// Initialize minio client object.
+	minioClient, err := minio.New(S3_HOST, &minio.Options{
+		Creds:  credentials.NewStaticV4(ACCESS_KEY_ID, SECRET_ACCESS_KEY, ""),
+		Secure: false,
+	})
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println(minioClient)
```

Comment:
Can we use "github.com/ipfs/go-log/v2" logs instead of `fmt.Println`? It is super straightforward to use, let me know if you need an example of how to use it.